src/main/java/com/google/error_analyzer/backend/StoreLogs.java

```diff
@@ -52,6 +53,7 @@ public String checkAndStoreLog(HttpServletRequest request, String fileName,
             indexName = getUniqueIndexName(indexName);
             final String response = storeLog(
                 request, indexName, log, OFFSET_FOR_PLAIN_TEXT );
+            findErrorsInIndex(indexName);
```

Comment:
Have we tested (UT) that the error index is created?

Reply:
test added